pkg/cli/importimage/importimage_test.go

```diff
@@ -597,11 +651,39 @@ func TestCreateImageImport(t *testing.T) {
 			o.InsecureFlagProvided = true
 		}
 
+		// set default for referencePolicy
+		if len(test.referencePolicy) == 0 {
+			for i := range test.expectedImages {
+				test.expectedImages[i].ReferencePolicy.Type = imagev1.SourceTagReferencePolicy
```

Comment:
I think the main reason for doing this here was to emulate the behavior of the user not needing to specify the default for it to be set, as it does when working with oc. These needed to be set because the Validate() step ends up mutating the ImportOptions object. But it would be nicer to not have to change the `expected`  fields ever.